lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -103,6 +138,18 @@
     });
   };
 
+  // Makes a reasonable determination that
+  // the data passed to the function is formatted
+  // JSON and returns `true`, if so. False, if not
+
+  function isJSON (json, callback) {
+    if (!json) {
+      return callback(new Error('You forgot to pass JSON data as the first argument.'));
+    }
+    var cases = [ true, false, null, '"', '-', '[', '{' ];
+    return cases.some(function(element) { return json[0] === element; });
```

Comment:
json[0] is going to be `'t'`, not `true`, so the test will fail for that case as written so far.

As for `.some`: back to IE9, so OK.